src/main/java/org/fundacionjala/coding/nestor/Customer.java

```diff
@@ -0,0 +1,77 @@
+package org.fundacionjala.coding.nestor;
+
+import java.util.Enumeration;
+import java.util.Vector;
+
+public class Customer {
+    private String _name;
+    private Vector<Rental> _rentals = new Vector<Rental>();
+
+    public Customer(String name) {
+        _name = name;
+    }
+
+    public void addRental(Rental arg) {
+        _rentals.addElement(arg);
+    }
+
+    public String getName() {
+        return _name;
+    }
+
+	public double getTotalAmount() {
+        double totalAmount = 0;
+        Enumeration<Rental> rentals = _rentals.elements();
+
+        while (rentals.hasMoreElements()) {
+            Rental element = (Rental) rentals.nextElement();
+            totalAmount += element.getCharge();
+        }
+
+        return totalAmount;
+    }
+    
+    public String statement() {
+        double totalAmount = 0;
+        int frequentRenterPoints = 0;
+        Enumeration<Rental> rentals = _rentals.elements();
+        String result = "Rental Record for " + getName() + "\n";
+        while (rentals.hasMoreElements()) {
+            double thisAmount = 0;
+            Rental each = (Rental) rentals.nextElement();
+            //determine amounts for each line
+            /*switch (each.getMovie().getPriceCode()) {
+                case Movie.REGULAR:
+                    thisAmount += 2;
+                    if (each.getDaysRented() > 2)
+                        thisAmount += (each.getDaysRented() - 2) * 1.5;
+                    break;
+                case Movie.NEW_RELEASE:
+                    thisAmount += each.getDaysRented() * 3;
+                    break;
+                case Movie.CHILDRENS:
+                    thisAmount += 1.5;
+                    if (each.getDaysRented() > 3)
+                        thisAmount += (each.getDaysRented() - 3) * 1.5;
+                    break;
+            }*/
+            // add frequent renter points
+            frequentRenterPoints++;
+            // add bonus for a two day new release rental
+            /*if ((each.getMovie().getPriceCode() == Movie.NEW_RELEASE)
+                    &&
+                    each.getDaysRented() > 1) frequentRenterPoints++;
+            //show figures for this rental
+            result += "\t" + each.getMovie().getTitle() + "\t" +
+                    String.valueOf(thisAmount) + "\n";
+            totalAmount += thisAmount;*/
+        }
+        //add footer lines
+        result += "Amount owed is " + String.valueOf(totalAmount) +
+                "\n";
+        result += "You earned " + String.valueOf(frequentRenterPoints)
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Directly append the argument of String.valueOf(). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1153)